.github/workflows/dm_ci.yml

```diff
@@ -0,0 +1,22 @@
+name: Continuous integration
+
+on: [push]
```

Comment:
Could we make it also on pull requests? 
Like:
`  pull_request:
    branches: [ master ]
`
Or maybe this is already included as you do that for each push

Reply:
And maybe (but it might be you have already activated this), we could also make that passing the test is  mandatory before merging a PR: https://stackoverflow.com/questions/60776412/github-actions-is-there-a-way-to-make-it-mandatory-for-pull-request-to-merge

Reply:
@fedenanni Thanks for the comments. Both are implemented now. If you don't have further comments, I will merge once the checks are passed.